.github/workflows/security_scans.yml

```
@@ -0,0 +1,22 @@
+name: SAST Snyk Code
+on:
+  schedule:
+    - cron: '23 14 * * *'
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

There is no `actions/checkout@master` - the branch is main, but the recommended version is `v5`.

```suggestion
      - uses: actions/checkout@v5
```

---

Added